### DISCUSSION TITLE
is there any react-typescript open project?

### DISCUSSION THREAD

**Discussion by YahefuYusufu** [EddieHubCommunity/support#6419] [Q&A]
Hi guys,my name is Yusuf, from sweden , i am looking for a open source project to improve my skills ,i am available for any kind of project (js,react,typescript)

**Comment by sansarmaske** [comment]
We do have [LinkFree](https://github.com/EddieHubCommunity/LinkFree) which uses [Next.js](https://nextjs.org), [MongoDB](https://www.mongodb.com) and [Tailwind CSS](https://tailwindcss.com). You can start there.

**Reply by YahefuYusufu** [reply]
thanks.

**Comment by Panquesito7** [comment]
Here's the website for a great project, which is built with [React](https://reactjs.org/), TypeScript, and [Next.js](https://nextjs.org/): https://github.com/TheAlgorithms/website

**Reply by YahefuYusufu** [reply]
thanks.

**Comment by raghu-007** [comment]
@YusufYakup, please do check this project 
https://github.com/mui/material-ui

**Reply by YahefuYusufu** [reply]
thanks man

**Comment by dinxsh** [comment]
yo @YahefuYusufu, would def recommend to check this articles, got some really cool react projects listed ranging from easy to hard difficulty

- [link 1](https://flatlogic.com/blog/best-react-open-source-projects/)
- [link 2](https://medium.mybridge.co/22-amazing-open-source-react-projects-cb8230ec719f)
- [link 3](https://javascript.plainenglish.io/7-react-open-source-project-to-inspire-you-806a898fc245)

meta opensource - [link](https://opensource.fb.com/projects/react/)

besides that of course [LinkFree](https://github.com/EddieHubCommunity/LinkFree) is p good to work on as well!

hope this helps!

**Reply by YahefuYusufu** [reply]
thanks bro
